<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QPushButton" name="listenButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Listen</string>
      </property>
     </widget>
     <widget class="QLabel" name="messageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>561</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Student Id:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="studentIdLineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999999999</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scoreLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>000</string>
      </property>
     </widget>
     <widget class="QComboBox" name="genderComboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>91</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSubmit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listenButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onListen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onListen()</slot>
  <slot>onSubmit()</slot>
 </slots>
</ui>
